2023-03-12 17:37:57.884765: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2023-03-12 17:38:00.068391: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2023-03-12 17:38:00.068998: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-03-12 17:38:23.200544: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/rds/general/user/mw1919/home/FYPDeepDeconstructedLearning/main.py", line 18, in <module>
    ppo.train(SpaceInvaders, SimpleNetwork, save_location = f'{os.environ["PBS_O_WORKDIR"]}/ppo_3')
  File "/rds/general/user/mw1919/home/FYPDeepDeconstructedLearning/src/algorithms/PPO.py", line 285, in train
    trainer = PPO.Trainer(game, network)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/rds/general/user/mw1919/home/FYPDeepDeconstructedLearning/src/algorithms/PPO.py", line 92, in __init__
    self.model = PPO.Model(
                 ^^^^^^^^^
AttributeError: type object 'PPO' has no attribute 'Model'
